adapters/adnuntius/adnuntiustest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,149 @@
+                                    },
+                                    "text": {},
+                                    "choices": {},
+                                    "clickUrl": "https://ads.adnuntius.delivery/c/n6xZq-VhchUD8PbfezltPoCESuAfspN-w0wjkkwzHXsAAAAQCtjQz9kbGWD4nuZy3q6HaHGLVNv__f6bBEfVP76Mfd6iokhXOQf7XpiOka60HChCnQjfT3AAqAO1swCIxQR73xnw2Okk_huqOKANzHxOy9gOZB8wEKOW8bs-Po5kTzB8U8UaB4BF9d48j6zHV7ST6-6cRBhpWFyKZUaUDEgmwKXM8xlQC-OJY7F9UJ0rN8OZQis1rnmeXrbik9VvxuV8Owsd_I9ZMS4G",
+                                    "urls": {
+                                        "url": "https://ads.adnuntius.delivery/c/n6xZq-VhchUD8PbfezltPoCESuAfspN-w0wjkkwzHXsAAAAQCtjQz9kbGWD4nuZy3q6HaHGLVNv__f6bBEfVP76Mfd6iokhXOQf7XpiOka60HChCnQjfT3AAqAO1swCIxQR73xnw2Okk_huqOKANzHxOy9gOZB8wEKOW8bs-Po5kTzB8U8UaB4BF9d48j6zHV7ST6-6cRBhpWFyKZUaUDEgmwKXM8xlQC-OJY7F9UJ0rN8OZQis1rnmeXrbik9VvxuV8Owsd_I9ZMS4G?ct=2587&r=http%3A%2F%2Fwww.google.com"
+                                    },
+                                    "urlsEsc": {
+                                        "url": "https%3A%2F%2Fads.adnuntius.delivery%2Fc%2Fn6xZq-VhchUD8PbfezltPoCESuAfspN-w0wjkkwzHXsAAAAQCtjQz9kbGWD4nuZy3q6HaHGLVNv__f6bBEfVP76Mfd6iokhXOQf7XpiOka60HChCnQjfT3AAqAO1swCIxQR73xnw2Okk_huqOKANzHxOy9gOZB8wEKOW8bs-Po5kTzB8U8UaB4BF9d48j6zHV7ST6-6cRBhpWFyKZUaUDEgmwKXM8xlQC-OJY7F9UJ0rN8OZQis1rnmeXrbik9VvxuV8Owsd_I9ZMS4G%3Fct%3D2587%26r%3Dhttp%253A%252F%252Fwww.google.com"
+                                    },
+                                    "destinationUrls": {
+                                        "url": "http://www.google.com"
+                                    },
+                                    "cpm": {
+                                        "amount": 20000.0,
+                                        "currency": "NOK"
+                                    },
+                                    "bid": {
+                                        "amount": 20.0,
+                                        "currency": "NOK"
+                                    },
+                                    "cost": {
+                                        "amount": 20.0,
+                                        "currency": "NOK"
+                                    },
+                                    "impressionTrackingUrls": [],
+                                    "impressionTrackingUrlsEsc": [],
+                                    "adId": "adn-id-1559784094",
+                                    "selectedColumn": "0",
+                                    "selectedColumnPosition": "0",
+                                    "renderedPixel": "https://ads.adnuntius.delivery/b/n6xZq-VhchUD8PbfezltPoCESuAfspN-w0wjkkwzHXsAAAAQCtjQz9kbGWD4nuZy3q6HaHGLVNv__f6bBEfVP76Mfd6iokhXOQf7XpiOka60HChCnQjfT3AAqAO1swCIxQR73xnw2Okk_huqOKANzHxOy9gOZB8wEKOW8bs-Po5kTzB8U8UaB4BF9d48j6zHV7ST6-6cRBhpWFyKZUaUDEgmwKXM8xlQC-OJY7F9UJ0rN8OZQis1rnmeXrbik9VvxuV8Owsd_I9ZMS4G.html",
```

Comment:
Just a suggestion: You could probably simplify your tests quite a bit by paring down what you're returning in your `mockResponse` as most of this isn't used by your adapter.